app/business/repositories/IAuthRepository.scala

```diff
@@ -0,0 +1,29 @@
+package business.repositories
+
+import business.models._
+import common.LastError
+
+import scala.concurrent.Future
+
+/**
+ * Created by liviuignat on 05/04/15.
+ */
+trait IAuthRepository {

```

Comment:
do you actually need the separation? there is only one class that implements this. (not to mention you're using .Net notations in Scala) :)